test/functional/ui/screen_concealing_spec.lua

```diff
@@ -0,0 +1,144 @@
+local helpers = require('test.functional.helpers')
```

Comment:
I think these would fit in `highlight_spec.lua`, or else please rename this to `highlight_conceal_spec.lua`.